sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableServiceImpl.java

```diff
@@ -547,6 +551,39 @@ public CompletionStage<MutateRowResponse> writeRecord(KV<ByteString, Iterable<Mu
 
       RowMutationEntry entry = RowMutationEntry.createFromMutationUnsafe(record.getKey(), mutation);
 
+      ServiceCallMetric serviceCallMetric = createServiceCallMetric();
```

Comment:
should this be called per element? I would imagine that they should get initialized once and incremented per eelemnt

Reply:
It is in the current flow. It may not need to be created every time